roles/kube-node-common/tasks/kubelet.yml

```diff
@@ -67,18 +67,28 @@
     repo: "deb https://apt.kubernetes.io/ kubernetes-{{ kubernetes_repo_distribution }} main"
     state: present
 
-- name: Making sure kubelet, kubectl, kubeadm, cri-tools are installed
+- name: Making sure kubelet, kubectl, kubeadm are installed
   apt:
     name:
       - openssl
       - kubelet={{ kubelet_version }}-00
       - kubeadm={{ kubeadm_version }}-00
       - kubectl={{ kubectl_version }}-00
-      - cri-tools={{ critools_version }}-00
       - kubernetes-cni={{ kubernetescni_version }}-00
     state: present
     update_cache: yes
 
+- name: Install cri-tools
+  become: yes
+  unarchive:
+    src: https://github.com/kubernetes-sigs/cri-tools/releases/download/v{{ critools_version }}/{{ item }}-v{{ critools_version }}-linux-amd64.tar.gz
```

Comment:
Why are we switching away from the DEB package? Since the upstream project provides them, could we check the sha256 of the downloaded archive?

Reply:
cri-tools was stuck at version 1.10 in Debian Kubernetes repositories. The only way to have all Kubernetes binaries aligned was to download cri-tools from GitHub releases. 

There is also a pending refactor to move all Kubernetes binaries to be installed from GitHub releases instead of Debian repositories. 

https://trello.com/c/ULhqwYTM

About the sha256 validation, sure! let me see how to do it!

Reply:
Added sha256 validation while downloading cri-tools packages ;)

Reply:
Pipeline passed, image verified (manually)

Reply:
I am OK with installing critools from binary but I would rather avoid installing Kubernetes binaries in the same way for the following 3 reasons:
- package integrity verification,
- drift from upstream documentation (see https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/),
- RPM/DEB packages do some than just shipping some binaries, they setup systemd unit and provide pre/post installation script that could ease version upgrades.

Anyway, I am open to discussion about this topic.

Reply:
Completely agree with you. Let's install from binaries those packages where is impossible to install from apt/rpm repositories. This is the cri-tools.

Reply:
Added this conversation to: https://trello.com/c/ULhqwYTM

Reply:
I will resolve this conversation, at least, for this PR ;) We can continue talking in Slack/Trello